src/main/java/com/synopsys/integration/detect/Application.java

```diff
@@ -83,152 +80,157 @@ public static boolean shouldExit() {
         return SHOULD_EXIT;
     }
 
-    public static void setShouldExit(final boolean shouldExit) {
+    public static void setShouldExit(boolean shouldExit) {
         SHOULD_EXIT = shouldExit;
     }
 
-    public static void main(final String[] args) {
-        final SpringApplicationBuilder builder = new SpringApplicationBuilder(Application.class);
+    public static void main(String[] args) {
+        SpringApplicationBuilder builder = new SpringApplicationBuilder(Application.class);
         builder.logStartupInfo(false);
         builder.run(args);
     }
 
     @Override
-    public void run(final ApplicationArguments applicationArguments) {
-        final long startTime = System.currentTimeMillis();
+    public void run(ApplicationArguments applicationArguments) {
+        long startTime = System.currentTimeMillis();
 
         //Events, Status and Exit Codes are required even if boot fails.
-        final EventSystem eventSystem = new EventSystem();
-        final DetectStatusManager statusManager = new DetectStatusManager(eventSystem);
+        EventSystem eventSystem = new EventSystem();
+        DetectStatusManager statusManager = new DetectStatusManager(eventSystem);
 
-        final ExitCodeUtility exitCodeUtility = new ExitCodeUtility();
-        final ExitCodeManager exitCodeManager = new ExitCodeManager(eventSystem, exitCodeUtility);
+        ExitCodeUtility exitCodeUtility = new ExitCodeUtility();
+        ExitCodeManager exitCodeManager = new ExitCodeManager(eventSystem, exitCodeUtility);
```

Comment:
Now that we have exit manager, could we rename exit code manager? Would something like ExitCodeListener make sense?